src/Workspaces/Remote/Core/Serialization/MessagePackFormatters.cs

```diff
@@ -24,6 +26,7 @@ internal sealed class MessagePackFormatters
     {
         internal static readonly ImmutableArray<IMessagePackFormatter> Formatters = ImmutableArray.Create<IMessagePackFormatter>(
             ProjectIdFormatter.Instance,
+            EncodingFormatter.Instance,
```

Comment:
nit since outside the compiler codebase: I didn't follow how the added tests cover/reach this new formatter. Could you clarify?

Reply:
It's picked up here: https://github.com/dotnet/roslyn/pull/64625/files#diff-1f47e0ff785a4203cc26c25920a79f2565ad31e7a050b7a4faa002a324a3a6f0R149